scripts/postinstall.js

```diff
@@ -44,25 +59,50 @@ const downloadPath = path.join(__dirname, '..', 'dbs');
 
 if (!fs.existsSync(downloadPath)) fs.mkdirSync(downloadPath);
 
-const download = (url) =>
-  new Promise((resolve) => {
-    https.get(url, (response) => {
-      resolve(response.pipe(zlib.createGunzip({})));
-    });
+const request = async (url, options) => {
+  const response = await new Promise((resolve, reject) => {
+    https
+      .request(
+        url,
+        {
+          auth: accountId ? `${accountId}:${licenseKey}` : null,
+          ...options,
+        },
+        (response) => {
+          resolve(response);
+        }
+      )
+      .on('error', (err) => reject(err))
+      .end();
   });
 
+  if (
```

Comment:
I wonder if we should switch over to `fetch` with nicer promise-based API and simple `{ redirect: 'follow' }` way of handling redirects

Reply:
I did actually switch to `fetch` at one stage just to get it working, haha. But I thought this would be exceeding the scope of what was necessary here.

I'm happy to make another PR for that change